### DISCUSSION TITLE
`rollup` configuration

### DISCUSSION THREAD

**Discussion by algomaster99** [renovatebot/renovate#18259] [Request Help] [answered]
Hi @rarkins ! Thanks for your talk at CHAINS workshop. I asked about the rollup feature and you said I could look at "rollup" option in the tool. I found https://github.com/renovatebot/renovate/issues/4404 which is still open. Do you have plans to make it configurable in the renovate-bot on GitHub actions? Let me know how I can give it a try for now.

**Comment by rarkins** [comment, marked as answer]
I think there was a confusion sorry, the rollup feature is not implemented yet

**Reply by algomaster99** [reply]
That is fine. Looking forward to the feature :)
